CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+This project is open to contributors, and we'd like to make it as easy as possible to share your ideas, and your code.
+
+Firstly, no contribution is too small. If you see something that's not right, please feel free to submit a spontaneous PR. It will very likely be reviewed same day.
+
+## Where to start
+
+If you don't know where to start, have a look at the list of [issues](https://github.com/tomarrell/lbadd/issues). If you find something interesting, feel free to say so in a comment, and you can be assigned to the issue.
+
+If someone is already assigned to an issue, feel free to mention that you're interested as well. Most issues can likely be broken down into more fine-grained tasks for newcomers to contribute to.
+
+If you don't find something in the issues, don't worry! There is **plenty** that is *likely not listed there*. If you think you have a clear idea about something that should be done, feel free to open an issue yourself and start a discussion. A maintainer will get back to you, and most likely assign you to the task or clarify some requirements beforehand.
```

Comment:
Maybe link the slack channel.

Should we allow issues that call for work? Should we maybe pin an issue that acts as megathread for work-seeking contributors?

Reply:
I think issues as a proposal mechanism should be fine. We can have a label to filter out the noise and mark them as such. A megathread might be a little hard to keep track of.

Reply:
I'm afraid that people might not always have an idea what they could work on. I have a ton of cards in the parser projects in the backlog, that are not issues, but noone will see them. There are tasks, but not all of them are issues and/or discoverable.

I just tried to formulate my megathread idea, and it won't work... Should we maybe add a new issue label `work wanted` or something, so that people can open an issue if they want to contribute but don't know what to work on?

Reply:
That sounds like the best way to go for now. We can always revise this if we need to in the future. I'll add a note to the doc.